frontend/apps/crates/entry/jig/play/src/player/sidebar/actions.rs

```diff
@@ -15,19 +17,14 @@ use super::state::State;
 pub fn send_report(state: Rc<State>) {
     state.player_state.loader.load(clone!(state => async move {
         let report_type = state.report_type.lock_ref();
-        log::info!("Sending report: {:?}", report_type);
-        // TODO: actually send report
         state.report_status.set(ReportStatus::Sent);
-
-        // TODO: enable once we actually send the reports
-        // spawn_local(clone!(state => async move {
-        //     TimeoutFuture::new(5_000).await;
-        //     let mut report_status = state.report_status.lock_mut();
-        //     // only update if status hasn't changed
-        //     if *report_status == ReportStatus::Sent {
-        //         *report_status = ReportStatus::Default;
-        //     }
-        // }));
+        spawn_local(clone!(state => async move {
+            TimeoutFuture::new(5_000).await;
+            let mut report_status = state.report_status.lock_mut();
+            if *report_status == ReportStatus::Sent {
+                *report_status = ReportStatus::Default;
+            }
+        }));
```

Comment:
Instead of this placeholder logic, you'll want to call the [`/jig/{id}/report`](https://github.com/ji-devs/ji-cloud/blob/sandbox/shared/rust/src/api/endpoints/jig/report.rs#L14-L29) endpoint:

```rust
let response = jig::report::Create::api_with_auth(CreateJigReport {
    // .. populate fields from state
}).await;
match response {
    Err(_err) => {
    },
    Ok(response) => {

    }
}
```

Also, I'm not sure whether the report status should be reset. If the student/teacher has reported it, it shouldn't be necessary to re-report it. They could refresh the page and report it again if they really wanted to.